packages/common/src/events/_internal/PropertyMapper.ts

```diff
@@ -94,11 +94,19 @@ export class EventPropertyMapper extends PropertyMapper<
 
     const heroActions: HubActionLink[] = [];
     if (store.allowRegistration) {
+      const eventHasEnded = obj.endDateTime < new Date();
```

Comment:
Given that the `arcgis-hub-event-about.tsx` component will need to know if the event occurred in the past in order to render one of the `calcite-notice` per [this comment](https://github.com/ArcGIS/opendata-ui/pull/13599#discussion_r1622878118), it may be better to define an additional `isPast` property on the `IHubEvent` POJO and set it's value to the result of this expression somewhere higher in this method outside this if-condition (maybe after assigning `obj.isRemoved`). That would eliminate the need for us to duplicate this date comparison logic here and in that component.